samples/CuriePME.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2017, Intel Corporation.
+
+// Test code to use the Curie Pattern Matching Engine within the Intel Curie Compute
+// on the Arduino 101/tinyTile, which uses the CuriePME API to train and classify data
+console.log("Curie Pattern Matching Engine test...");
+
+// import pme module
+var CuriePME = require("pme");
```

Comment:
Why not just call it pme? :) This makes it look like (in the code sample) it is a special object which is always accessible in